### PR TITLE
Updated kops version and patch

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*
* Updated to v1.26.0-beta.2. We will have to update it again when the non-beta version is out
* Updated patch to be able to apply to current version.
  * Also made changes so we don't have to update some insignificant things in the patch every time we update it

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/kubernetes/kops/patches/0001-remove-hardcoded-requierment-on-flannel-plugin.patch [hunk 1]
```diff
@@ -1,28 +1,26 @@
-From 2ec5afbedada5688c13aa8e8d9ac9ffba3ae4511 Mon Sep 17 00:00:00 2001
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```
Comment (TerryHowe) [inline projects/kubernetes/kops/patches/0001-remove-hardcoded-requierment-on-flannel-plugin.patch:1]: Is this allowed? Like a wildcard or something?

Comment (kschumy) [reply projects/kubernetes/kops/patches/0001-remove-hardcoded-requierment-on-flannel-plugin.patch:1]: We do it off the EKS-D Kubernetes patches

**Pull request merged**